### ISSUE TITLE
Is there a way to get execution ResultSummary

### ISSUE THREAD

**Issue by calltherain** [neo4j-labs/neo4rs#161]
I'm migrating some of our Golang code to Rust, in Golang Neo4j library, I can [get the result summary](https://neo4j.com/docs/go-manual/current/result-summary/#_query_counters)  like nodes created/deleted, relationship added/removed, attributes updated...

Looks like this is not possible in neo4rs at the moment? 

Thanks

**Comment by knutwalker** [comment]
You're right, we don't support result summary yet. We're going to improve the support for additional Bolt features soon and I'll keep that in mind (since the counters are a protocol feature and not part of a response itself).
